app/models/subscription.rb

```diff
@@ -116,8 +116,9 @@ def validate_external_id
     return unless active?
 
     # NOTE: We want unique external id per organization.
-    used_ids = organization.subscriptions.active.pluck(:external_id)
-    errors.add(:external_id, :value_already_exist) if used_ids&.include?(external_id)
+    if organization.subscriptions.active.exists?(external_id:)
```

Comment:
Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`. (https://rubystyle.guide#if-as-a-modifier)